telemetry/event_counters.go

```diff
@@ -5,9 +5,31 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/telemetry"
 	"github.com/hashicorp/go-metrics"
+	"github.com/pokt-network/smt"
+
+	prooftypes "github.com/pokt-network/poktroll/x/proof/types"
+)
+
+const (
+	eventTypeMetricKey = "event_type"
 )
 
-const eventSuccessKey = "event_type"
+type ClaimProofStage = string
```

Comment:
OPTIONAL.

I looked into enum best practices in golang and found this: https://yourbasic.org/golang/iota/#complete-enum-type-with-strings-best-practice

Up to you if it's how we do things

Reply:
I'm familiar with the convention, it just seemed unnecessarily verbose, especially in this case.